(MainThread) 190902_19:40:47.874 DEBUG Trading.py:133 now is 2019-09-02 19:40:47.874053
(MainThread) 190902_19:40:47.879 ERROR wrapper.py:56 ERROR -1 2104 Market data farm connection is OK:usfarm.nj
(MainThread) 190902_19:40:47.879 ERROR wrapper.py:56 ERROR -1 2104 Market data farm connection is OK:eufarm
(MainThread) 190902_19:40:47.879 ERROR wrapper.py:56 ERROR -1 2104 Market data farm connection is OK:cashfarm
(MainThread) 190902_19:40:47.879 ERROR wrapper.py:56 ERROR -1 2104 Market data farm connection is OK:usfarm
(MainThread) 190902_19:40:47.880 ERROR wrapper.py:56 ERROR -1 2106 HMDS data farm connection is OK:euhmds
(MainThread) 190902_19:40:47.881 ERROR wrapper.py:56 ERROR -1 2106 HMDS data farm connection is OK:cashhmds
(MainThread) 190902_19:40:47.883 ERROR wrapper.py:56 ERROR -1 2106 HMDS data farm connection is OK:fundfarm
(MainThread) 190902_19:40:47.883 ERROR wrapper.py:56 ERROR -1 2106 HMDS data farm connection is OK:ushmds
(Thread-1) 190902_19:40:48.059 ERROR reader.py:52 unhandled exception in EReader thread
Traceback (most recent call last):
  File "d:\py\tms\ibapi\reader.py", line 34, in run
    data = self.conn.recvMsg()
  File "d:\py\tms\ibapi\connection.py", line 99, in recvMsg
    buf = self._recvAllMsg()
  File "d:\py\tms\ibapi\connection.py", line 119, in _recvAllMsg
    buf = self.socket.recv(4096)
OSError: [WinError 10038] An operation was attempted on something that is not a socket
